tendrl/ceph_integration/objects/pool/__init__.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+from tendrl.commons.etcdobj.etcdobj import EtcdObj
+from tendrl.ceph_integration import objects
+
+LOG = logging.getLogger(__name__)
+
+
+class Pool(objects.CephIntegrationBaseObject):
+    def __init__(self, cluster_id=None, pool_id=None,
+                 poolname=None, pg_num=None, min_size=None,
+                 used=None, percent_used=None, *args, **kwargs):
+        super(Pool, self).__init__(*args, **kwargs)
+
+        self.value = 'clusters/%s/Pools/%s'
+        self.cluster_id = cluster_id
```

Comment:
store/get cluster_id and fsid from tendrl_ns.tendrl_context.cluster_id/fsid wherever required. remove self.cluster_id